pytest-{{cookiecutter.plugin_name}}/.gitignore

```diff
@@ -59,6 +59,9 @@ instance/
 # Sphinx documentation
 docs/_build/
 
+# MkDocs documentation
+/site/
```

Comment:
depends on what are the chances of someone actually having a directory called "site" somewhere else in the project - without the initial slash it will ignore all of them.

I'll change it to "site/" if you think it's safe enough.